bot/build.sh

```diff
@@ -287,6 +295,9 @@ else
     TARBALL_STEP_ARGS+=("--resume" "${REMOVAL_TMPDIR}")
 fi
 
+# Make sure we define storage, so that the TMPDIR is set to this in eessi_container.sh
+TARBALL_STEP_ARGS+=("--storage" "${STORAGE}")
```

Comment:
This should not be necessary, because we should resume from a previous temporary directory. `--storage` tells `eessi_container.sh` "instead of using a default location for temporary storage such as `$TMPDIR` or simply `/tmp`, use `$STORAGE`"

Reply:
Hm, I'm trying to figure out why I made this change, but don't see it. I'll first make sure I get a successful build again in https://github.com/EESSI/software-layer/pull/912 and then remove it, and try again.

Reply:
Initial build https://github.com/EESSI/software-layer/pull/912#issuecomment-2654300700 succeeded. Now, implemented your suggested change and trying again https://github.com/EESSI/software-layer/pull/912#issuecomment-2654421669